templates/inspec/tests/integration/configuration/mm-attributes.yml

```diff
@@ -183,4 +183,7 @@ bigquery_table:
   table_id: inspec_gcp_bigquery_table
   description: A BigQuery table
   expiration_time: 1738882264000
-  time_partitioning_type: DAY
\ No newline at end of file
+  time_partitioning_type: DAY
+
+repository:
+  name: inspec-gcp-repository
```

Comment:
I think repositories are pretty new in Terraform. Do we have a version in-mind for users when they run Terraform for testing?

(I know we're probably the only people running Terraform for testing, but it would be a good idea to state somewhere what version of Terraform users should use to run all of this)

Reply:
Good idea. I should add this to the readme in the inspec-gcp repo. They were already using Terraform for testing, but this addition may require people to upgrade. There is no terraform version stated currently